dm/syncer/validator_checkpoint.go

```diff
@@ -482,10 +480,10 @@ func (c *validatorPersistHelper) loadTableStatus(tctx *tcontext.Context) (map[st
 	return res, nil
 }
 
-func (c *validatorPersistHelper) loadErrorCount() (map[pb.ValidateErrorState]int64, error) {
+func (c *validatorPersistHelper) loadErrorCount(tctx *tcontext.Context) (map[pb.ValidateErrorState]int64, error) {
```

Comment:
As we are supposed to use a different context from the validator's, is the `tctx` of `validatorPersistHelper` still useful? It seems it's only used for logging.